src/modules/storage/LedgerStorage.ts

```diff
@@ -583,20 +590,13 @@ export class LedgerStorage extends Storages {
      * is called with an error.
      */
     public putBlocks(block: Block): Promise<void> {
-        const genesis_timestamp: number = this.genesis_timestamp;
         const now_timestamp = Math.floor(new Date().getTime() / 1000);
-
-        // In the production environment, it should be treated as the received time.
-        if (process.env.NODE_ENV === "production") {
-            block.header.time_offset = now_timestamp - genesis_timestamp;
-        } else {
-            block.header.time_offset = JSBI.toNumber(block.header.height.value) * this.block_interval;
-        }
-
+        block.header.time_offset = now_timestamp - this.genesis_timestamp;
```

Comment:
What will this time_offset be used for?

Reply:
I think it should also be set to the calculated offset based on the block height.